sample.env

```diff
@@ -0,0 +1,4 @@
+GOOGLE_CLIENT_ID=client_id
```

Comment:
Even that's only demo ENV vars, I think it's worth a clear naming, ala: google_client_id, google_client_secret, ...